MediaBrowser.XbmcMetadata/Parsers/BaseNfoParser.cs

```diff
@@ -1136,20 +1134,20 @@ private void FetchFromRatingsNode(XmlReader reader, T item)
                     switch (reader.Name)
                     {
                         case "rating":
-                        {
-                            if (reader.IsEmptyElement)
                             {
-                                reader.Read();
-                                continue;
-                            }
+                                if (reader.IsEmptyElement)
+                                {
+                                    reader.Read();
+                                    continue;
+                                }
 
-                            var ratingName = reader.GetAttribute("name");
+                                var ratingName = reader.GetAttribute("name");
 
-                            using var subtree = reader.ReadSubtree();
-                            FetchFromRatingNode(subtree, item, ratingName);
+                                using var subtree = reader.ReadSubtree();
+                                FetchFromRatingNode(subtree, item, ratingName);
 
-                            break;
-                        }
+                                break;
+                            }
```

Comment:
This just seems to change indentation?

Reply:
Correct but seems like now it's as it should be? At least all other switch statements are formatted like this too.

Reply:
The other statements are incorrectly formatted